UOP1_Project/Assets/Scripts/Sound/SoundEmitter.cs

```diff
@@ -0,0 +1,58 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.Audio;
+
+[RequireComponent (typeof(AudioSource))]
+public class SoundEmitter : MonoBehaviour
+{
+	private AudioSource thisAudioSource;
```

Comment:
We do actually: [Open Projects - Conventions](https://docs.google.com/document/d/1-eUWZ0lWREFu5iH-ggofwnixDDQqalOoT4Yc0NpWR3k/edit)

Should be renamed to _audioSource.

Reply:
Sorry, since we did not have code guidelines I just defaulted to what I use on my personal projects, its fixed on the latest commit for audioSource. Thanks!

Reply:
Thanks for the like davejrodriguez, I will read it and update all code that doesn't follow the convention as soon as possible.